src/components/Form/Field.js

```diff
@@ -75,6 +75,10 @@ export const FormField = ({
       ? stepData[data.name][recurringIndex][name]
       : stepData[name];
 
+  if (field?.currencies && field?.currencies !== undefined) {
```

Comment:
undefined is falsey, why have you added this as an additional check to this statement?